akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -194,6 +194,8 @@ object Replicator {
     Props(new Replicator(settings)).withDeploy(Deploy.local).withDispatcher(settings.dispatcher)
   }
 
+  val DefaultMajorityMinCap: Int = 2
```

Comment:
Yes if it's 0 will be more clear, but the result would be the same. I'll change it!